ddl/get.lua

```diff
@@ -66,11 +67,24 @@ local function get_metadata(space_name, metadata_name)
 end
 
 local function get_sharding_func(space_name)
-    local record = get_metadata(space_name, "sharding_func")
+    local record = cache.internal.get(space_name)
+
     if not record then
         return nil
     end
 
+    return record.processed
+end
+
+local function get_sharding_func_raw(space_name)
+    local record = cache.internal.get(space_name)
+
+    if not record or not record.raw then
+        return nil
+    end
```

Comment:
While we'll always get either tuple or `nil`, it seems natural to look, whether the cache contains an error, or explicitly clarify that the error corresponds only to `processed` and not `raw` in the cache structure description, or comment why exactly we ignore the error from the cache here.

Reply:
fixed, comment added